news/2020-11-27-project-loved-november-2020.md

```diff
@@ -0,0 +1,220 @@
+
+[Virtual Riot - Shindeiru](https://osu.ppy.sh/beatmapsets/866120#mania) by [Gekido-](https://osu.ppy.sh/users/4693052)\
+150 BPM, 2:34 | \[4K\] 4.67★
+
+Omae wa mo shindeiru. An iconic phrase that has beget many memes, jokes, and in this case, songs and charts. One such song is Virtual Riot's *Shindeiru*, charted by lemonguy (formerly Gekido-). lemonguy is no stranger to the Loved section, having been nominated for many popular and impressive tech and LN charts alike, such as [Chik Habit](https://osu.ppy.sh/beatmapsets/732982) and [The Clockwork Rose](https://osu.ppy.sh/beatmapsets/740979). This pattern continues with *Shindeiru*, an extremely technical and bursty file that will catch you off-guard if you aren't fully ready for the speed and difficulty of the patterns throughout. Outside of the bursts there are also some extended jack patterns that can be difficult to transition in and out of, especially with the one-handed trills that go along with them. It is a difficult and satisfying chart, but if you aren't up to the challenge, you may already be dead.
+
+[![](/wiki/shared/news/2020-11-27-project-loved-november-2020/mania/2-tornado.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Camellia - Tornado](https://osu.ppy.sh/beatmapsets/619276#mania) by [inteliser](https://osu.ppy.sh/users/1824775)\
+175 BPM, 4:29 | \[4K\] 6.03★, \[4K\] 6.06★
+
+The storm is back, and it's hitting harder than ever. In this month's edition of Project Loved, we bring to you *Tornado* by inteliser, a heavy, masterfully crafted chart and a relic of its time. As a fairly well-known mapper in the community, inteliser is certainly well acquainted with the Loved section, with maps like [Mirage](https://osu.ppy.sh/beatmapsets/486877) and [Dodger's Gadget (UNMIXED)](https://osu.ppy.sh/beatmapsets/727666) -- though this map particularly strays away from the LN-based direction that players are more accustomed to in inteliser's Loved maps. This map almost entirely tests physicality and technicality -- spanning no less than four minutes, this map barely gives any breaks, filled to the brim with extremely difficult and dense jack and burst patterns clocking in at 175 BPM. This map has been featured in multiple large tournaments, including the finals pool of MWC 2020. As a challenging stamina-intensive chart, its persistent popularity among competitive players throughout the years has shown that it has stood the test of time and well deserves its spot in the Loved section. TO-TO-TO-TORNADO!
+
+[![](/wiki/shared/news/2020-11-27-project-loved-november-2020/mania/3-super-eurobeat-for-running-selections-vol-2.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Various Artists - SUPER EUROBEAT FOR RUNNING SELECTIONS VOL. 2](https://osu.ppy.sh/beatmapsets/587484#mania) by [Jole](https://osu.ppy.sh/users/2883132)\
```

Comment:
looks like this should have the 0 star template thing excluded

Reply:
fixed